pages/art-pieces/[slug].js

```diff
@@ -0,0 +1,26 @@
+import ArtPieceDetails from "@/components/ArtPieceDetails";
+import { useRouter } from "next/router";
+
+export default function ArtPieceDetailsPage({ pieces }) {
+  const router = useRouter();
+  const { slug } = router.query;
+
+  const selectedArtPiece = pieces.find((piece) => piece.slug === slug);
+
+  return (
+    <>
+      <ArtPieceDetails
+        image={selectedArtPiece.imageSource}
+        title={selectedArtPiece.title}
+        artist={selectedArtPiece.artist}
+        year={selectedArtPiece.year}
+        genre={selectedArtPiece.genre}
+      />
+      ;
+    </>
```

Comment:
Vielleicht könnte man den Code hier verkürzen wie folgt?

```suggestion
      <ArtPieceDetails
        image={selectedArtPiece.imageSource}
        title={selectedArtPiece.title}
        artist={selectedArtPiece.artist}
        year={selectedArtPiece.year}
        genre={selectedArtPiece.genre}
      />